jekyll/_docs/enterprise/advanced-configuration.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: enterprise
+title: Advanced Configuration
+category: [resources]
+order: 0
+description: "Advance Configurations"
+hide: false
+---
+
+
+# Advanced Configuration 
+
+## Proxy Support
+
+
+### Getting Started
+
+For the purposes of this demo and to follow the examples, we assume that we have an unauthenticated http proxy at `10.0.0.33:3128` and the services box is at `10.0.1.238`. We use `ghe.example.com` as our github enterprise host.
+
+In an ideal case, traffic to S3 will not be proxied, and instead be added to the NO_PROXY rules (to bypass it out add `s3.amazonaws.com,*.s3.amazonaws.com`  to `NO_PROXY`)
+
+Everything here must be done **before** installing CircleCI on fresh vm's/instances.
+
```

Comment:
It's not clear to me if this applies just to amazon proxy settings or also installations that are behind a corporate proxy. In the corporate case, adding s3 to `NO_PROXY` probably won't work since all internet traffic is typically routed through the proxy, or not at all.

If it's only amazon proxies, you may want to add corporate proxy install instructions as well. Working around the lack of proxy support in some of the scripts was the hardest part of the install process for me.